<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanerUARTSetupWindow</class>
 <widget class="QDialog" name="ScanerUARTSetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="COMPortComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>COM-порт:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="connectBtn">
     <property name="text">
      <string>Подключиться</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>SSID сети Wi-Fi</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Пароль сети Wi-Fi:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Имя сканера:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ssidInput"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="passInput"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="scanerNameInput"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
